src/basic/generate-af-list.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 
-cpp -dM -include sys/socket.h - </dev/null | \
+$1 -E -dM -include sys/socket.h - </dev/null | \
```

Comment:
In some cases I need to pass both the compiler and a list or arguments, see e.g. `generate-cap-list.sh`. So passing the compiler as "$@" wouldn't work in the general case. I don't worry too much about the compiler path having spaces.